v1/transform.yaml

```diff
@@ -48,6 +48,13 @@ paths:
           description: Normalise the DOM to yield cleaner wikitext?
           type: boolean
           required: false
+        - name: if-match
+          in: header
+          description: |
+            An `ETag` header of the original render indicating it's revision and timeuuid.
+            Required if the `title` and `revision` parameters are present.
```

Comment:
Perhaps more precisely, to emphasize that it's not an `or`: "Required if both the `title` and `revision` parameters are present."